### PR TITLE
feat: Implement design spec part1

### PR DESCRIPTION
**Pull request by bluekurinji**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
...
+
+Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+
+## Running end-to-end tests
+
+Run `ng e2e` to execute the end-to-end tests via a platform of your choice. To use this command, you need to first add a package that implements end-to-end testing capabilities.
+
+## Further help
+
+To get more help on the Angular CLI use `ng help` or go check out the [Angular CLI Overview and Command Reference](https://angular.io/cli) page.
```
Comment (chrisolsen) [inline README.md]: Can you remove these changes? We need to keep the previous README file

File: angular.json [hunk 2]
```diff
@@ -25,13 +18,12 @@
...
             "polyfills": "src/polyfills.ts",
             "tsConfig": "tsconfig.app.json",
-            "inlineStyleLanguage": "scss",
             "assets": [
               "src/favicon.ico",
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/styles.css"
```
Comment (chrisolsen) [inline angular.json]: This and the other changes to this file looks like things are being changed from scss to css. I have a feeling most other teams prefer scss, so I think these changes need to be removed

File: package.json [hunk 3]
```diff
@@ -10,32 +10,31 @@
...
-    "@angular/common": "~13.1.0",
-    "@angular/compiler": "~13.1.0",
-    "@angular/core": "~13.1.0",
-    "@angular/forms": "~13.1.0",
-    "@angular/platform-browser": "~13.1.0",
-    "@angular/platform-browser-dynamic": "~13.1.0",
-    "@angular/router": "~13.1.0",
-    "rxjs": "~7.4.0",
+    "@abgov/styles": "^1.0.0-alpha.13",
+    "@abgov/web-components": "^1.0.0-alpha.74",
```
Comment (chrisolsen) [inline package.json]: Let's keep the newer versions :+1:

File: src/app/app.component.ts [hunk 4]
```diff
@@ -3,8 +3,13 @@ import { Component } from '@angular/core';
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
-  styleUrls: ['./app.component.scss']
+  styleUrls: ['./app.component.css'],
```
Comment (chrisolsen) [inline src/app/app.component.ts]: Let's keep the scss

File: README.md (hunk 1 above)
Comment (bluekurinji) [reply README.md]: done

File: angular.json (hunk 2 above)
Comment (bluekurinji) [reply angular.json]: done

File: package.json (hunk 3 above)
Comment (bluekurinji) [reply package.json]: done

File: src/app/app.component.ts (hunk 4 above)
Comment (bluekurinji) [reply src/app/app.component.ts]: done

**Pull request merged**